pycrdt_websocket/django_channels/yroom_storage.py

```diff
@@ -0,0 +1,187 @@
+import time
+from typing import Optional
+
+import redis.asyncio as redis
+from pycrdt import Doc
+
+
+class BaseYRoomStorage:
```

Comment:
Maybe this should be an abstract base class, with a dummy implementation that does nothing?

Reply:
Agree!